features/app/components/AbstractApp.js

```diff
@@ -1,5 +1,14 @@
 import React, { Component } from 'react';
+
 import { roomSet } from '../../base/conference';
+import {
+    disposeLib,
+    initLib
```

Comment:
If we're talking about destroying/disposing/removing stuff, then we have following actions:
- destroyLocalTracks
- removeLocalParticipant
- disposeLib
- disconnect (for connection+conference)
so I'm not sure if we should rename them. I'd say, that current names reflect the meaning of what's going on in the best way. E.g. "destroyLib" is not a very good name - it shouldn't destroy anything, it should just do a cleanup of stuff initialized with JitsiMeetJS.init().